controllers/api_controller.js

```diff
@@ -13,48 +19,67 @@ exports.getTopics = (req, res, next) => {
 };
 
 exports.getApi = (req, res, next) => {
-  res
-    .status(200)
-    .send(endPoints)
+  res.status(200).send(endPoints);
 };
 
 exports.getArticleById = (req, res, next) => {
   const { article_id } = req.params;
   fetchArticleById(article_id)
-  .then((article) => {
-    if (!article) {
-      return Promise.reject({status: 404, msg: "Article not found" });
-    }
-    res.status(200).send({ article });
-  })
-  .catch((err) => {
-    next(err)
-  })
+    .then((article) => {
+      if (!article) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
+      }
+      res.status(200).send({ article });
+    })
+    .catch((err) => {
+      next(err);
+    });
 };
 
 exports.getArticles = (req, res, next) => {
-fetchArticles()
-.then((articles) => {
-    res.status(200).send({articles})
-})
-.catch((err) => {
-    next(err)
-})
-}
+  fetchArticles()
+    .then((articles) => {
+      res.status(200).send({ articles });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
 
 exports.getCommentsByArticleId = (req, res, next) => {
- const {article_id} = req.params
- checkArticleExist(article_id)
- .then((exists) => {
-    if(!exists) {
-        return Promise.reject({status: 404, msg: 'Article not found'})
-    }
-    return fetchCommentsByArticleId(article_id)
- })
- .then ((comments) => {
-    res.status(200).send({comments})
- })
- .catch((err) => {
-    next(err)
- })
-}
+  const { article_id } = req.params;
+  checkArticleExist(article_id)
+    .then((exists) => {
+      if (!exists) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
+      }
+      return fetchCommentsByArticleId(article_id);
+    })
+    .then((comments) => {
+      res.status(200).send({ comments });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.addComments = (req, res, next) => {
+  const { article_id } = req.params;
+  const { body, username } = req.body;
+  if (!username || !body) {
+    return res.status(400).send({ msg: "Missing required field" });
+  }
+  checkArticleExist(article_id)
```

Comment:
article_id is also a foreign key so again, a separate check that the article exists should not be necessary as a psql error will be thrown.